docker-compose.yaml

```diff
@@ -1,17 +1,19 @@
 services:
-  dev:
+  base: &default
     image: gfw/pipe-segment
     build:
       context: .
-      dockerfile: Dockerfile-scheduler
+      dockerfile: Dockerfile
+
+  dev:
+    <<: *default
+    entrypoint: "./scripts/run.sh"
     volumes:
       - ".:/opt/project"
       - "gcp:/root/.config/"
+
   gcloud:
-    image: gfw/pipe-segment
-    build:
-      context: .
-      dockerfile: Dockerfile-scheduler
+    <<: *default
     entrypoint: gcloud
     volumes:
       - "gcp:/root/.config/"
```

Comment:
I don't know why this was set up this way. You don't need our custom image to run these gcloud commands. You can instead use [the official google cloud cli dockerfile](https://hub.docker.com/r/google/cloud-sdk/). We want this service to generate authorizations, but we don't need the full blown python environment for it. That way, you also don't need to replicate the base image.

Reply:
I'm going to tackle this in this same PR. I think it makes sense.

Reply:
Done